docs/notebooks/plot_slopes.ipynb

```diff
@@ -0,0 +1,2503 @@
+{
```

Comment:
<p>I'm not sure I understand this sentence: </p><p> </p><blockquote>Additionally, interactions can be specified for combinations <strong>and</strong> multiple categorical and continuous variables; making the interpretation of the the slopes more difficult</blockquote><p> </p><p>let's chat in our call about it.</p><p> </p><p>The second paragraph could be omitted, since it says the same than the previous section. An option to omitting the paragraph is to make it a sentence or two.</p><p> </p><p>All the rest looks good, the only thing I would like to talk about is the following </p><p> </p><blockquote>With GLMs, every covariate essentially interacts with itself</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/701/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>agreed</p>

Reply:
Regarding

> Additionally, interactions can be specified for combinations and multiple categorical and continuous variables; making the interpretation of the the slopes more difficult

I am implying that interaction can be specified for continuous variables, categorical variables, or between continuous and categorical variables. But after reading this a second time, I see how this is confusing. I will re-word it.

Reply:
Regarding 

> With GLMs, every covariate essentially interacts with itself...

I get this information from Richard McElreath's Statistical Rethinking on pgs. 325-327 (which I will need to cite if we decide to keep this paragraph). I wanted to include this because it emphasises the importance of visualizing predictions made by the model (in particular for GLMs)